AnagramSolver.Cli/Program.cs

```diff
@@ -1,10 +1,53 @@
-﻿namespace AnagramSolver.Cli
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Threading.Tasks;
+
+namespace AnagramSolver.Cli
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var builder = new ConfigurationBuilder()
+               .AddJsonFile($"appsettings.json", true, true);
+            var config = builder.Build();
+            var minLength = config["MinWordLength"];
+            int minLengthAsNumber = Int32.Parse(minLength);
+            var maxAnagrams = config["MaxNumberOfAnagrams"];
+            int maxAnagramsAsNumber = Int32.Parse(maxAnagrams);
+            Console.WriteLine(maxAnagramsAsNumber);
+
+            var result = new BusinessLogic.Classes.AnagramSolver();
+            var command = "";
```

Comment:
I would prefer instead of quotes `""` to use `string.Empty` or maybe in your case we can set up `string command = null`

Reply:
done